README.md

```diff
@@ -63,6 +63,8 @@ Producer
 * `USER_CRT` - the user's certificate
 * `USER_KEY` - the user's private key
 * `LOG_LEVEL` - logging level  
+* `PRODUCER_ACKS` = acknowledgement level
+* `ADDITIONAL_CONFIG` = additional configuration for producer
```

Comment:
You should add some explanation abotu the format - that it is a key=value map etc.